spec/02-integration/03-db/07-tags_spec.lua

```diff
@@ -448,6 +448,7 @@ for _, strategy in helpers.each_strategy() do
     end
     func("trigger defined for table", function()
       for entity_name, dao in pairs(db.daos) do
```

Comment:
Would prefer `for _, dao in ...` here to avoid reassignment

Reply:
via https://github.com/Kong/kong/pull/9182/commits/86da43bd7545b146178c05c834099a6168f4cc6c